### PR TITLE
Easing functions

### PR DESCRIPTION
**Pull request by Softizo**
I'll test the easing functions after this PR.

### CODE REVIEW

File: little/src/ease.rs [hunk 1]
```diff
@@ -0,0 +1,231 @@
+#![feature(core_intrinsics)]
```
Comment (Thomas-QM) [inline little/src/ease.rs]: feature flags go in the lib.rs file, ya dont need em for every file :^)

File: little/src/ease.rs [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+
+extern crate core;
```
Comment (Thomas-QM) [inline little/src/ease.rs]: same with extern crate

File: little/src/ease.rs [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+    let mut td = t / d;
+    
+    if t == 0.0 { b }
+    else if td == 1.0 { b + c }
+    else {
+        let p = d * 0.3;
+        let a = c;
+        let s = p / 4.0;
+        td -= 1.0;
+        let postfix = a * 2.0f32.powf(10.0 * td);
```
Comment (Thomas-QM) [inline little/src/ease.rs]: are you sure powf works? intrinsics uses separate module functions, powf is for the std::f32 iirc... idk just run ``cargo check``

Comment (Softizo) [reply little/src/ease.rs]: It works

File: little/src/ease.rs (hunk 2 above)
Comment (Softizo) [reply little/src/ease.rs]: Got it :)

File: little/src/ease.rs (hunk 1 above)
Comment (Softizo) [reply little/src/ease.rs]: :))

**Comment by Thomas-QM** [comment]
owo

**Comment by Thomas-QM** [comment]
waut fuk

**Comment by Thomas-QM** [comment]
OwO

**Comment by Softizo** [comment]
lol

**Pull request merged**